src/main/resources/com/linecorp/armeria/server/docs/assets/js/armeria.js

```diff
@@ -215,8 +240,22 @@ $(function () {
     var fragment = URI(window.location.href).fragment(true);
     var fragmentParams = fragment.query(true);
     if (fragmentParams.req) {
-      debugText.val(fragmentParams.req);
-      submitDebugRequest();
+      var req = JSON.parse(fragmentParams.req);
+
+      debugText.val(req.args);
+
+      if ('http_headers' in req) {
+        debugHttpHeadersText.val(req.http_headers);
+      } else {
+        debugHttpHeadersText.val(''); // Remove the default value if set
+      }
+
+      functionContainer.find('.debug-submit').focus();
```

Comment:
How about focusing the text area instead of the submit button? A user might press 'space' or 'enter' key by mistake.